### PR TITLE
Checklist for jazzband adoption

### LINKED ISSUE #113: Implement Jazzband guidelines for django-fsm-log
This issue tracks the implementation of the [Jazzband guidelines](https://jazzband.co/about/guidelines) for the project [django-fsm-log](https://github.com/jazzband/django-fsm-log)

It was initiated by @ticosax who was automatically assigned in addition to the [Jazzband roadies](https://jazzband.co/roadies).

See the [TODO list](#todos) below for the generally required tasks, but feel free to update it in case the project requires it.

Feel free to ping a [Jazzband roadie](https://jazzband.co/roadies) if you have any question.

## TODOs<a name="todos"></a>

- [x] Fix all links in the docs (and README file etc) from old to new repo
- [x] Add the [Jazzband badge](https://jazzband.co/about/guidelines#badges) to the `README` file
- [x] Add the [Jazzband contributing guideline](https://jazzband.co/about/guidelines#contributing-guidelines) to the `CONTRIBUTING.md` or `CONTRIBUTING.rst` file
- [x] Check if continuous testing works with [GitHub Actions](https://github.com/features/actions)
- [x] Check if test coverage is tracked with [Codecov](https://codecov.io)
- [x] Add `jazzband` account to [PyPI](https://pypi.org/) project as maintainer role (e.g. URL: https://pypi.org/manage/project/django-fsm-log/collaboration/)
- [x] Add `jazzband-bot` as maintainer to the [Read the Docs](https://readthedocs.org/) project (e.g. URL: https://readthedocs.org/dashboard/django-fsm-log/users/)
- [x] Create Sphinx based docs using the README as source.
- [x] Add incoming GitHub webhook integration to Read the Docs project (e.g. URL: https://readthedocs.org/dashboard/django-fsm-log/integrations/)
- [x] Fix project URL in [GitHub project](https://github.com/jazzband/django-fsm-log) description
- [x] Review project if other services are used and port them to Jazzband
- [x] Decide who is project lead for the project (if at all)
- [ ] Set up CI for Jazzband project releases if needed and open ticket if yes

## Project details

<table>
  <caption></caption>
  <tr>
    <td>Description</td>
    <td>Automatic logging for Django FSM</td>
  </tr>
  <tr>
    <td>Homepage</td>
    <td></td>
  </tr>
  <tr>
    <td>Stargazers</td>
    <td>146</td>
  </tr>
  <tr>
    <td>Open issues</td>
    <td>10</td>
  </tr>
  <tr>
    <td>Forks</td>
    <td>63</td>
  </tr>
  <tr>
    <td>Default branch</td>
    <td>master</td>
  </tr>
  <tr>
    <td>Is a fork</td>
    <td>False</td>
  </tr>
  <tr>
    <td>Has Wiki</td>
    <td>True</td>
  </tr>
  <tr>
    <td>Has Pages</td>
    <td>False</td>
  </tr>
</table>

### PR DESCRIPTION
**Pull request by ticosax**
fixes #113

### CODE REVIEW

**Comment by MRigal** [comment]
@ticosax You may add a `.pre-commit-config.yaml` File to fix the pre-commit.ci check. Are you familiar with it? Else I could help

**Comment by ticosax** [comment]
> @ticosax You may add a `.pre-commit-config.yaml` File to fix the pre-commit.ci check. Are you familiar with it? Else I could help

I'm not familiar with it, I'm welcoming your help. Thanks

**Comment by ticosax** [comment]
@MRigal I would like to share the leadership on this project with you if you'd like. Feel free to make a request.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
 
-[![test suite](https://github.com/gizmag/django-fsm-log/actions/workflows/test_suite.yml/badge.svg)](https://github.com/gizmag/django-fsm-log/actions/workflows/test_suite.yml)
-[![Code Health](https://landscape.io/github/gizmag/django-fsm-log/master/landscape.png)](https://landscape.io/github/gizmag/django-fsm-log/master)
-[![codecov](https://codecov.io/gh/gizmag/django-fsm-log/branch/master/graph/badge.svg)](https://codecov.io/gh/gizmag/django-fsm-log)
+[![test suite](https://github.com/jazzband/django-fsm-log/actions/workflows/test_suite.yml/badge.svg)](https://github.com/jazzband/django-fsm-log/actions/workflows/test_suite.yml)
+[![codecov](https://codecov.io/gh/jazzband/django-fsm-log/branch/master/graph/badge.svg)](https://codecov.io/gh/jazzband/django-fsm-log)
+[![Jazzband](https://jazzband.co/static/img/badge.svg)](https://jazzband.co/)
+[![Documentation Status](https://readthedocs.org/projects/django-fsm-log/badge/?version=latest)](https://django-fsm-log.readthedocs.io/en/latest/?badge=latest)
+
 Automatic logging for the excellent [Django FSM](https://github.com/kmmbvnr/django-fsm)
```
Comment (MRigal) [inline README.md:8]: Although not strictly related, the original package is now located at https://github.com/viewflow/django-fsm (and we need to make a PR there too :-D)

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+.. include:: ../README.md
+   :parser: myst_parser.sphinx_
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```
Comment (MRigal) [inline docs/index.rst:21]: Unfortunately I have no experience in setting up RTD, but I think when I look at https://django-fsm-log--114.org.readthedocs.build/en/114/ that `modindex` does not and won't exist

**Review by MRigal — APPROVED** [review]
Fantastic work @ticosax !

I have only two small mini requests. Plus a general note that I would add a link to the readthedocs.io in the "About" section of the repo

**Pull request merged**